keps/sig-network/563-dual-stack/README.md

```diff
@@ -1460,6 +1439,7 @@ of this feature?**
 * **Will enabling / using this feature result in any new calls to the cloud 
 provider?**
   No
+  [TODO] Do we have any metrics around success/failure of cluster IP allocation? Number of dual stack services? Pod IP allocation failures?
```

Comment:
I'm not sure if this is the right place for this information. If we don't have the ability to surface these metrics, should we say so?

Reply:
Errors are surfaced when the user makes a call. We don't focus on metrics-server at this point.